docs/strategy-callbacks.md

```diff
@@ -318,11 +318,11 @@ class AwesomeStrategy(IStrategy):
 
         # evaluate highest to lowest, so that highest possible stop is used
         if current_profit > 0.40:
-            return stoploss_from_open(0.25, current_profit, is_short=trade.is_short, leverage=trade.leverage)
+            return stoploss_from_open(0.25, current_profit, is_short=trade.is_short)
```

Comment:
Why would we remove this? it's in the function and people SHOULD use it if they use leverage (and want to avoid problems). It also doesn't hurt non-leveraged trades - as the value will be 1.

So i don't see why we should remove it from the docs -  the "new" docs are now misleading ... 

https://github.com/freqtrade/freqtrade/blob/caf524c68584995e5ef6e4d2ba165ca4279e264f/freqtrade/strategy/strategy_helper.py#L86-L91

Reply:
ah, seems I checked on the wrong branch which wasn't rebased on current dev. sorry